src/services.js

```diff
@@ -122,6 +122,26 @@ const tilesetStatus = function (tilesetId) {
   setTimeout(checkStatus, 10000, tilesetId);
 };
 
+// Check job status and report warnings
+const jobStatus = async function (tilesetId, jobId) {
+  console.log("Checking completed job status...");
+  const response = await mtsService.tilesetJob({ tilesetId: `${process.env.MTS_USERNAME}.${tilesetId}`, jobId }).send();
+  return response.body;
+};
+
+const checkJobWarnings = async function (tilesetId, job) {
+  console.log("Checking job warnings");
+  const status = await jobStatus(tilesetId, job);
+  console.log(status);
+  console.log(status.warnings);
```

Comment:
Do we need to log `status.warnings` here, or does the following if block handle it?